Tasks/Npm/npmtoolrunner.ts

```diff
@@ -29,17 +32,27 @@ export class NpmToolRunner extends tr.ToolRunner {
     public exec(options?: tr.IExecOptions): Q.Promise<number> {
         options = this._prepareNpmEnvironment(options) as tr.IExecOptions;
 
-        return super.exec(options).catch((reason: any) => {
-            return this._printDebugLog(this._getDebugLogPath(options)).then((value: void): number => {
-                throw reason;
-            });
-        });
+        this._saveProjectNpmrc();
```

Comment:
Is removing and restoring the file is the only way to make npm ignore project .npmrc?